app/assets/config/manifest.js

```diff
@@ -0,0 +1,11 @@
+//= link_tree ../images
+//= link_tree ../builds
+
+//= link_tree ../javascripts .js
+
+//= link codemirror/modes/liquid.js
```

Comment:
I think we need this despite the `link_tree ../javascripts .js` line, because this file is in `vendor/assets/javascripts`, and not in `app/assets/javascripts`.